src/posts/2021-09-06-using-css-frameworks.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "CSS Frameworks: Tailwind- en Windi CSS"
+category: inspiration
+tags: framework, css, sass. frontend
```

Comment:
```suggestion
tags: framework, css, sass, frontend
```

Reply:
Still getting used to my new (large) keyboard ;-)